conf/application.conf

```diff
@@ -27,7 +27,7 @@ play.filters.csp {
     style-src   = "'self' 'unsafe-inline' https://www.googletagmanager.com https://tagmanager.google.com https://fonts.googleapis.com"
     img-src     = "'self' https://ssl.gstatic.com www.gstatic.com https://www.google-analytics.com data: https://region1.google-analytics.com https://region1.analytics.google.com https://*.google-analytics.com https://*.analytics.google.com https://fonts.gstatic.com https://www.googletagmanager.com"
     font-src    = "'self' https://ssl.gstatic.com www.gstatic.com https://fonts.gstatic.com https://fonts.googleapis.com"
-    connect-src = "raw.githubusercontent.com https://region1.google-analytics.com https://region1.analytics.google.com https://*.google-analytics.com https://*.analytics.google.com"
+    connect-src = "'self' 'unsafe-inline' raw.githubusercontent.com https://region1.google-analytics.com https://region1.analytics.google.com https://*.google-analytics.com https://*.analytics.google.com"
```

Comment:
As matter of interest, why the `unsafe-inline` is needed here?

Reply:
forget to remove, thanks. not needed this